build.sbt

```diff
@@ -88,3 +88,13 @@ lazy val samples = project
   )
 
 lazy val tools = project in file("tools")
+
+lazy val docs = project
+  .in(file("docs"))
+  .enablePlugins(ParadoxPlugin)
+  .settings(
+    paradoxTheme := Some(builtinParadoxTheme("generic")),
+    paradoxProperties in Compile ++= Map(
+      "github.base_url" -> "https://github.com/kornilova-l/scala-native-bindgen/tree/master/"
```

Comment:
It should be something like `s"https://github.com/kornilova-l/scala-native-bindgen/tree/v${version.value}"`
But what will be the way to release new version then?
* manually change `version := "0.2-SNAPSHOT"` to `version := "0.2"`
* generate docs
* release
* manually change `version := "0.2"` to `version := "0.3-SNAPSHOT"`

Is there a way to automate this?